### DISCUSSION TITLE
GTAIV.EFLC.Various.Fixes v1.51

### DISCUSSION THREAD

**Discussion by valentyn-l** [valentyn-l/GTAIV.EFLC.Various.Fixes#7] [Announcements]
Mod version 1.51. Please consult the mod's changelog for detailed list of changes.  Also check out our page on [GTA Forums](https://gtaforums.com/topic/975211-various-fixes/) for more updates!

<hr /><em>This discussion was created from the release <a href='https://github.com/valentyn-l/GTAIV.EFLC.Various.Fixes/releases/tag/'>GTAIV.EFLC.Various.Fixes v1.51</a>.</em>

**Comment by Punisher11995** [comment]
You wouldn't happen to know where the platypus cargo ship textures are stored? I've been trying to fix the PS3 version and haven't been able to find the poster from the intro that is next to Dave. Any help is appreciated. your mod is great btw.

**Reply by valentyn-l** [reply]
The file you're looking for is called ruskgallposts.wdr. It's found inside of data\maps\interiors\generic\brownstones.img

**Reply by Punisher11995** [reply]
Thanks a ton!
